### PR TITLE
Nano

### PR DESCRIPTION
**Pull request by scopatz**

### CODE REVIEW

File: recipes/nano/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - autoconf
+    - make
+    - pkg-config
```
Comment (isuruf) [inline recipes/nano/meta.yaml]: Technically, the 3 packages above belong in `build`.

Comment (scopatz) [reply recipes/nano/meta.yaml]: Really?  That is annoying. I am not sure I fully understand the distinction between build and host, then.

**Comment by scopatz** [comment]
@conda-forge/staged-recipes this is R4R

File: recipes/nano/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/nano/meta.yaml]: In this example, they'll be windows packages and hence in `host`. There's some more detail at https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host why python should be in host when you are building python packages.

Comment (scopatz) [reply recipes/nano/meta.yaml]: I see thanks!

**Pull request merged**